docs/source/datasets/index.rst

```diff
@@ -461,4 +461,148 @@ The following columns are included in these datasets:
+       datasets. 
+   * - First name
+     - :code:`first_name`
+     - 
+   * - Middle initial
+     - :code:`middle_initial`
+     - 
+   * - Last name
+     - :code:`last_name`
+     - 
+   * - Age
+     - :code:`age`  
+     - Rounded down to an integer.
+   * - Date of birth
+     - :code:`date_of_birth`
+     - Formatted as YYYY-MM-DD.
+   * - Mailing address street number
+     - :code:`mailing_address_street_number`
+     - 
+   * - Mailing address street name
+     - :code:`mailing_address_street_name`
+     - 
+   * - Mailing address unit number
+     - :code:`mailing_address_unit_number`
+     - 
+   * - Mailing address PO box
+     - :code:`mailing_address_po_box`
+     - 
+   * - Mailing address city
+     - :code:`mailing_address_city`    
+     - 
+   * - Mailing address state
+     - :code:`mailing_address_state`  
+     - 
+   * - Mailing address ZIP code
+     - :code:`mailing_address_zipcode`
+     - 
+   * - Social security number 
+     - :code:`ssn`
+     - 
+   * - Individual tax identification number
+     - :code:`itin`
+     - 
+   * - Income 
+     - :code:`income`
+     - 
+   * - Employer ID 
+     - :code:`employer_id`
+     -  
+   * - Employer Name 
+     - :code:`employer_name`
+     - 
+   * - Employer street number
+     - :code:`employer_street_number`
+     - 
+   * - Employer street name
+     - :code:`employer_street_name`
+     - 
+   * - Employer unit number
+     - :code:`employer_unit_number`
+     - 
+   * - Employer city
+     - :code:`employer_city`    
+     - 
+   * - Employer state
+     - :code:`employer_state`  
+     - 
+   * - Employer ZIP code
+     - :code:`employer_zipcode`
+     - 
+   * - Joint filer first name
+     - :code:`spouse_first_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer middle initial
+     - :code:`spouse_middle_initial`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer last name
+     - :code:`spouse_last_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address street number
+     - :code:`spouse_mailing_address_street_number`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address street name
+     - :code:`spouse_mailing_address_street_name`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address unit number
+     - :code:`spouse_mailing_address_unit_number`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer mailing address PO box
+     - :code:`spouse_mailing_address_po_box`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer social security number
+     - :code:`spouse_ssn`
+     - Only included if the simulant has a joint filer (i.e., spouse).
+   * - Joint filer individual tax identification number
```

Comment:
Same comment as above -- ITINs should go in the SSN column and this column should be removed.